service/history/workflow/task_generator.go

```diff
@@ -35,6 +35,7 @@ import (
 	"go.temporal.io/api/serviceerror"
 
 	enumsspb "go.temporal.io/server/api/enums/v1"
+
```

Comment:
nit: this empty line can be remove, they are all server imports.

Reply:
Fixed, thanks!